packages/http/src/mocker/errors.ts

```diff
@@ -0,0 +1,8 @@
+import { Omit } from '@stoplight/types';
+import { ProblemJson } from '../types';
+
+export const INVALID_REQUEST_PAYLOAD: Omit<ProblemJson, 'detail'> = {
+  name: 'INVALID_REQUEST_PAYLOAD',
+  title: 'Invalid request body payload',
+  status: 422,
```

Comment:
Looks like you've swapped `400` to `422`. You think it matches better than `400`? If so maybe we could name the errors in a way that would reflect the status code?

How about `UNPROCESSABLE_ENTITY`?

Reply:
Makes sense. b5eb0ea